templates/huawei_vrp_display_version.textfsm

```diff
@@ -0,0 +1,9 @@
+Value VRP_VERSION (\S+)
+Value PRODUCT_VERSION (.+)
+Value MODEL (.+)

```

Comment:
```suggestion
Value PRODUCT_VERSION (\S+)
Value MODEL (\S+)
```
Keeping uptime as **.+** makes sense, but I think the rest should just match non-whitespace so they shouldn't catch other things.

```suggestion
^HUAWEI\s${MODEL}\suptime\sis\s${UPTIME}$$
```
This should help prevent catching any additional whitespaces and shows we're expecting end of line for this

Reply:
I use `.+` because `ATN 910C-B` in huawei_vrp_display_version2.raw have whitespace

Reply:
Sounds good. Thanks for the example.